conda_smithy/templates/main.yml.tmpl

```diff
@@ -0,0 +1,18 @@
+on:
+  status: {}
+  check_run:
+    types:
+      - completed
+
+jobs:
+  regro-cf-autotick-bot-action:
+    runs-on: ubuntu-latest
+    name: regro-cf-autotick-bot-action
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: regro-cf-autotick-bot-action
+        id: regro-cf-autotick-bot-action
+        uses: regro/cf-autotick-bot-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is jina2 going to replace the `{{` here? We need to check the output template because that would be wrong. 

I think maybe we should copy paster the template to the right spot.

Reply:
Can you manually cat the output file? I'd be more comfortable with a copy-paste.

Reply:
I'm not certain where it exists on disk, hence why I'm using jinja, since it seems to know.

Reply:
hmmmmm. In the test, let's add a call to print the file on disk and look. We can use [pkg_resources](https://setuptools.readthedocs.io/en/latest/pkg_resources.html) to grab the file too.

Reply:
For files that you want to copy over without any templating, you can put them in the `feedstock_content/` dir

Reply:
We want to pull it over only if automerge is turned on. I guess we will need to go back and delete the file later?